### PR TITLE
Add support new kb_elmo PCBs

### PR DESCRIPTION
**Pull request by kb-elmo**
## Description

Adds support for four previously uncommited PCBs that I designed some time ago.
Since they are all public on my Github and some people are using them for their projects now it was about time I added them here as well :)

Boards added:
- eliza 40 ergo
- bm42 drop-in
- dizzy40 drop-in
- gamehand pad

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kb_elmo/3liza/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        KC_TRNS, KC_LEFT, KC_DOWN, KC_RGHT, KC_PGDN, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,           KC_F12,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS,          KC_TRNS,          KC_TRNS,                   KC_TRNS,          KC_TRNS,          KC_TRNS
+    ),
+    [3] = LAYOUT(
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS,          KC_TRNS,          KC_TRNS,                   KC_TRNS,          KC_TRNS,          KC_TRNS
+    )
```
Comment (fauxpark) [inline keyboards/kb_elmo/3liza/keymaps/default/keymap.c:30]:
```suggestion
```

File: keyboards/kb_elmo/3liza/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        KC_TRNS, KC_LEFT, KC_DOWN, KC_RGHT, KC_PGDN, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,           KC_F12,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS,          KC_TRNS,          KC_TRNS,                   KC_TRNS,          KC_TRNS,          KC_TRNS
+    ),
+    [3] = LAYOUT(
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS,          KC_TRNS,          KC_TRNS,                   KC_TRNS,          KC_TRNS,          KC_TRNS
+    )
```
Comment (fauxpark) [inline keyboards/kb_elmo/3liza/keymaps/via/keymap.c:30]:
```suggestion
```

File: keyboards/kb_elmo/bm42/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        KC_TRNS, KC_LEFT, KC_DOWN, KC_RGHT, KC_PGDN, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,           KC_F12,
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,                   KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+    [3] = LAYOUT(
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,                   KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS
+    )
```
Comment (fauxpark) [inline keyboards/kb_elmo/bm42/keymaps/default/keymap.c:30]:
```suggestion
```

File: keyboards/kb_elmo/bm42/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        KC_TRNS, KC_LEFT, KC_DOWN, KC_RGHT, KC_PGDN, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,           KC_F12,
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,                   KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+    [3] = LAYOUT(
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,                   KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS
+    )
```
Comment (fauxpark) [inline keyboards/kb_elmo/bm42/keymaps/via/keymap.c:30]:
```suggestion
```

File: keyboards/kb_elmo/dizzy40/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        KC_TRNS, KC_LEFT, KC_DOWN, KC_RGHT, KC_PGDN, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,           KC_F12,
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,                   KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+    [3] = LAYOUT(
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,                   KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS
+    )
```
Comment (fauxpark) [inline keyboards/kb_elmo/dizzy40/keymaps/default/keymap.c:30]:
```suggestion
```

File: keyboards/kb_elmo/dizzy40/keymaps/via/keymap.c [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+        KC_TRNS, KC_LEFT, KC_DOWN, KC_RGHT, KC_PGDN, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,           KC_F12,
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,                   KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+    [3] = LAYOUT(
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,                   KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS
+    )
```
Comment (fauxpark) [inline keyboards/kb_elmo/dizzy40/keymaps/via/keymap.c:30]:
```suggestion
```

File: keyboards/kb_elmo/gamehand/info.json [hunk 7]
```diff
@@ -0,0 +1,62 @@
+{
+    "manufacturer": "kb-elmo",
+    "keyboard_name": "GameHand",
+    "url": "https://github.com/kb-elmo/gamehand",
+    "maintainer": "kb-elmo",
+    "usb": {
+        "vid": "0xA68C",
+        "pid": "0x4D90",
+        "device_version": "0.0.1",
+        "polling_interval": 1,
```
Comment (fauxpark) [inline keyboards/kb_elmo/gamehand/info.json:10]:
```suggestion
```

File: keyboards/kb_elmo/gamehand/keymaps/via/keymap.c [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS
+    ),
+    [3] = LAYOUT(
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS
+    )
```
Comment (fauxpark) [inline keyboards/kb_elmo/gamehand/keymaps/via/keymap.c:46]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/kb_elmo/gamehand/info.json (hunk 7 above)
Comment (fauxpark) [reply keyboards/kb_elmo/gamehand/info.json:10]: It's the default.

File: keyboards/kb_elmo/3liza/readme.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
+# 3liza
+
+![3liza](https://i.imgur.com/OGpPhRel.jpeg)
```
Comment (waffle87) [inline keyboards/kb_elmo/3liza/readme.md:3]:
```suggestion
![3liza](https://i.imgur.com/OGpPhRelh.jpeg)
```

File: keyboards/kb_elmo/3liza/readme.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+* **Bootmagic reset**: Hold down the top left key and plug in the keyboard
+* **Physical reset**: Briefly short the two contacts labeled "RESET" on the back of the PCB
+* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```
Comment (waffle87) [inline keyboards/kb_elmo/3liza/readme.md:28]:
```suggestion
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```

File: keyboards/kb_elmo/bm42/readme.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
+# BM42
+
+![BM42](https://i.imgur.com/DljDoaTl.jpeg)
```
Comment (waffle87) [inline keyboards/kb_elmo/bm42/readme.md:3]:
```suggestion
![BM42](https://i.imgur.com/DljDoaTlh.jpeg)
```

File: keyboards/kb_elmo/bm42/readme.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+* **Bootmagic reset**: Hold down the top left key and plug in the keyboard
+* **Physical reset**: Briefly short the two contacts labeled "RESET" on the back of the PCB
+* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```
Comment (waffle87) [inline keyboards/kb_elmo/bm42/readme.md:28]:
```suggestion
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```

File: keyboards/kb_elmo/dizzy40/readme.md [hunk 13]
```diff
@@ -0,0 +1,28 @@
+# Dizzy40
+
+![Dizzy40](https://i.imgur.com/TNXHaUGl.jpeg)
```
Comment (waffle87) [inline keyboards/kb_elmo/dizzy40/readme.md:3]:
```suggestion
![Dizzy40](https://i.imgur.com/TNXHaUGlh.jpeg)
```

File: keyboards/kb_elmo/dizzy40/readme.md [hunk 14]
```diff
@@ -0,0 +1,28 @@
...
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+* **Bootmagic reset**: Hold down the top left key and plug in the keyboard
+* **Physical reset**: Briefly short the two contacts labeled "RESET" on the back of the PCB
+* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```
Comment (waffle87) [inline keyboards/kb_elmo/dizzy40/readme.md:28]:
```suggestion
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```

File: keyboards/kb_elmo/gamehand/readme.md [hunk 15]
```diff
@@ -0,0 +1,28 @@
+# GameHand
+
+![gamehand](https://i.imgur.com/ia3bcUBl.jpg)
```
Comment (waffle87) [inline keyboards/kb_elmo/gamehand/readme.md:3]:
```suggestion
![gamehand](https://i.imgur.com/ia3bcUBlh.jpg)
```

File: keyboards/kb_elmo/gamehand/readme.md [hunk 16]
```diff
@@ -0,0 +1,28 @@
...
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+* **Bootmagic reset**: Hold down the top left key and plug in the keyboard
+* **Physical reset**: Briefly short the two contacts labeled "RESET" on the back of the PCB
+* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```
Comment (waffle87) [inline keyboards/kb_elmo/gamehand/readme.md:28]:
```suggestion
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```

File: keyboards/kb_elmo/3liza/readme.md (hunk 9 above)
Comment (fauxpark) [reply keyboards/kb_elmo/3liza/readme.md:3]: It's already thumbailed, you can change the `l` to an `h` if you want (max 1024px across).

**Review by waffle87 — APPROVED** [review]

**Comment by kb-elmo** [comment]
Any update on this? 😄

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by kb-elmo** [comment]
Thanks! 😄

**Pull request merged**